### PR TITLE
Fixes #67: Support custom APK bundle formats in `parseAppMeta()`

### LINKED ISSUE #67: Android: Support custom APK bundle formats in `parseAppMeta()`
Not terribly important, but since we now support installing custom APK bundle formats (#64), we should also support parsing metadata for them.

### PR DESCRIPTION
**Pull request by baltpeter**

### CODE REVIEW

**Review by zner0L — CHANGES_REQUESTED** [review]
Nice! I like that you took the time to implement it properly.

File: src/util.ts [hunk 1]
```diff
@@ -73,45 +76,98 @@ export const parseAppMeta = async (
...
+                }
+            }).then(bundle.close);
+            if (baseApkPath) return parseApk(baseApkPath);
+            return undefined;
+        }
+
+        return parseApk(appPath);
     } else if (platform === 'ios') {
-        const meta = await ipaInfo(appPath);
+        const meta = await ipaInfo(appPath as AppPath<'ios'>);
```
[the remark this answers is not in the document]
Comment (baltpeter) [reply src/util.ts:170]: The type is defined in the function definition. But TS doesn't understand that `if`ing on `platform` should narrow the type of `appPath`.

[the remark this answers is not in the document]
Comment (zner0L) [reply src/util.ts:157]: I don’t understand. I was talking about `ipaInfo(string)`, which should be `ipaInfo(AppPath<'ios'>)` instead.

[the remark this answers is not in the document]
Comment (baltpeter) [reply src/util.ts:157]: I mean sure, we can do that. But we'll still need the cast here.

[the remark this answers is not in the document]
Comment (zner0L) [reply src/util.ts:157]: Yes, I know. I liked the casting but it was unnecessary before. That’s what I was referring to.

**Pull request merged**